.kokoro/system-single.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# `-e` enables the script to automatically fail when a command fails
+# `-o pipefail` sets the exit code to non-zero if any command fails,
+# or zero if all commands in the pipeline exit successfully.
+set -eo pipefail
+
+pwd
+
+# If NOX_SESSION is set, it only runs the specified session,
+# otherwise run all the sessions.
+if [[ -n "${NOX_SESSION:-}" ]]; then
```

Comment:
nit: I don't think the `:-` gets you anything here (cf https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#Shell-Parameter-Expansion)


Also, what you have is fine, but you could equally well do 

```
SESSION_ARG=""
[[ -z "${NOX_SESSION} ]] || SESSION_ARG="-s ${NOX_SESSION}"
python3 -m nox ${SESSION_ARG}
```

This works in bash; in zsh, there's an additional tweak that's needed. I assume this file is executed, not sourced, in which case bash will always be used.

Reply:
Fixed in 68e60f2